frontend/webadmin/modules/frontend/src/main/resources/org/ovirt/engine/ui/frontend/AppErrors.properties

```diff
@@ -1545,6 +1545,9 @@ VM_POOL_CANNOT_REMOVE_RUNNING_VM_FROM_POOL=Cannot detach VM from VM-Pool. VM is
 VM_POOL_CANNOT_UPDATE_POOL_NOT_FOUND=Cannot edit VM-Pool properties - VM-Pool not found.
 VM_POOL_NOT_FOUND=VM-Pool not found
 ACTION_TYPE_FAILED_CLUSTER_IS_BEING_UPDATED=Cannot ${action} ${type}. Cluster ${clusterName} is being updated.
+ACTION_TYPE_FAILED_CLUSTER_UPGRADE_NOT_STARTED=Cannot update progress, cluster upgrade has not been started.
```

Comment:
It would be nice to include correlation ID and progress value.

Reply:
(will followup)